components/Footer.js

```diff
@@ -1,21 +1,53 @@
+import Link from "next/link";
 import footerStyles from "../styles/Footer.module.css";
 
 export default function Footer() {
   return (
     <footer className={footerStyles.footer}>
-      <a
-        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-        target="_blank"
-        rel="noopener noreferrer"
-        className={footerStyles.a}
-      >
-        Powered by{" "}
-        <img
-          src="/vercel.svg"
-          alt="Vercel Logo"
-          className={footerStyles.logo}
-        />
-      </a>
+      <div className={footerStyles.container}>
+        <div className={footerStyles.column}>
+          <ul className={footerStyles.footerList}>
+            <li className={footerStyles.footerItem}>
+              <Link href="/about-us">
+                <a>About Us</a>
+              </Link>
+            </li>
+            <li className={footerStyles.footerItem}>
+              <Link href="/blog">
+                <a>Blog</a>
+              </Link>
+            </li>
+            <li className={footerStyles.footerItem}>
+              <Link href="/contact-us">
+                <a>Contact Us</a>
+              </Link>
+            </li>
```

Comment:
We could create a constant with the following structure in a utils file or something.

```
const links = [
   { text: 'About Us',  href: '/about-us' },
   { text: 'Blog',  href: '/blog' },
   { text: 'Contact Us',  href: '/contact-us' },
]
```

then here we could do the following to remove some repeated code.

```suggestion
{links.map((link) => (
  <li className={footerStyles.footerItem} key={link.href}>
    <Link href={link.href}>
      <a>{link.text}</a>
    </Link>
  </li>
))}
```

Reply:
That's a great idea, Juliani and @rebecatschiedel ! If you feel comfortable about implementing this @rebecatschiedel , pls go ahead.

Reply:
I am on it!!
Thank you @Schlickmann !! Sometimes I forgot I can use logic inside HTML and CSS. 
My navbars will never be the same anymore!! =)